diff --git a/src/tpm2/TPMCmd/Platform/src/VendorInfo.c b/src/tpm2/TPMCmd/Platform/src/VendorInfo.c
index 972bfd44..0ef9ad19 100644
--- a/src/tpm2/TPMCmd/Platform/src/VendorInfo.c
+++ b/src/tpm2/TPMCmd/Platform/src/VendorInfo.c
@@ -10,22 +10,22 @@
 #include "Platform.h"
 
 // In this sample platform, these are compile time constants, but are not required to be.
-#define MANUFACTURER    "IBM"
-#define VENDOR_STRING_1 "SW  "
-#define VENDOR_STRING_2 " TPM"
+#define MANUFACTURER    "AMD\0"
+#define VENDOR_STRING_1 "AMD\0"
+#define VENDOR_STRING_2 "\0\0\0\0"
 #define VENDOR_STRING_3 "\0\0\0\0"
 #define VENDOR_STRING_4 "\0\0\0\0"
-#define FIRMWARE_V1     (0x20240125)
-#define FIRMWARE_V2     (0x00120000)
+#define FIRMWARE_V1     (0x00060020)
+#define FIRMWARE_V2     (0x00000006)
 #define MAX_SVN         255
 
-#if SIMULATION	// libtpms added
+#if SIMULATION  // libtpms added
 static uint32_t currentHash = FIRMWARE_V2;
 #endif
-static uint16_t currentSvn  = 10;
+static uint16_t currentSvn = 10;
 
 // Similar to the Core Library's ByteArrayToUint32, but usable in Platform code.
-static uint32_t StringToUint32(const char s[4])		// libtpms changed: added const
+static uint32_t StringToUint32(const char s[4])  // libtpms changed: added const
 {
     uint8_t* b = (uint8_t*)s;  // Avoid promotion to a signed integer type
     return (((uint32_t)b[0] << 8 | b[1]) << 8 | b[2]) << 8 | b[3];
@@ -86,7 +86,7 @@ LIB_EXPORT uint16_t _plat__GetTpmFirmwareMaxSvn(void)
 
 // Called by the simulator to set the TPM Firmware SVN reported by
 // getCapability.
-#if SIMULATION		// libtpms added
+#if SIMULATION  // libtpms added
 LIB_EXPORT void _plat__SetTpmFirmwareHash(uint32_t hash)
 {
     currentHash = hash;
@@ -98,7 +98,7 @@ LIB_EXPORT void _plat__SetTpmFirmwareSvn(uint16_t svn)
 {
     currentSvn = MIN(svn, MAX_SVN);
 }
-#endif			// libtpms added
+#endif  // libtpms added
 
 #if SVN_LIMITED_SUPPORT
 // Dummy implmenentation for obtaining a Firmware SVN Secret bound
