diff --git a/client/src/main.c b/client/src/main.c
index b07e25e7..e019f994 100644
--- a/client/src/main.c
+++ b/client/src/main.c
@@ -864,20 +864,9 @@ int main_frameThread(void * unused)
 
 static void checkUUID(void)
 {
-  if (!g_state.spiceReady || !g_state.guestUUIDValid)
-    return;
-
-  if (memcmp(g_state.spiceUUID, g_state.guestUUID,
-        sizeof(g_state.spiceUUID)) == 0)
-    return;
-
-  app_msgBox(
-      "SPICE Configuration Error",
-      "You have connected SPICE to the wrong guest.\n"
-      "Input will not function until this is corrected.");
-
-  g_params.useSpiceInput = false;
-  g_state.spiceClose = true;
-  purespice_disconnect();
+  // Spoofed Windows VM UUIDs
+  // Windows reports invalid UUIDs for VMs with duplicate or zeroed UUIDs
+  // so we have to ignore them here to be able to use clipboard and input
+  return;
 }
 